src/mono/wasm/build/WasmApp.LocalBuild.props

```diff
@@ -0,0 +1,51 @@
+<!--
+    WasmApp.LocalBuild* files are meant for projects that want to use
+    a local build of `dotnet/runtime` repo.
+
+    Required properties to be set:
+
+    $(RuntimeSrcDir)       - path to the runtime checkout. This should have already been
+                             built, and so should have `artifacts` directory.
+    $(RuntimeConfig)       - This is the config that the above checkout was built with.
+                             This can be different from the configuration for the user project.
+-->
+<Project>
+  <PropertyGroup>
+    <TargetArchitecture>wasm</TargetArchitecture>
+    <TargetOS>Browser</TargetOS>
+    <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
+    <_NetCoreAppToolCurrent>net5.0</_NetCoreAppToolCurrent>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(RuntimeSrcDir)' != '' and '$(WasmBuildSupportDir)' == ''">
+    <ArtifactsBinDir>$(RuntimeSrcDir)\artifacts\bin\</ArtifactsBinDir>
+    <EMSDK_PATH Condition="'$(EMSDK_PATH)' == ''">$(RuntimeSrcDir)\src\mono\wasm\emsdk\</EMSDK_PATH>
+
+    <MicrosoftNetCoreAppRuntimePackLocationToUse>$(ArtifactsBinDir)microsoft.netcore.app.runtime.browser-wasm\$(RuntimeConfig)\</MicrosoftNetCoreAppRuntimePackLocationToUse>
+
+    <WasmAppBuilderDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'WasmAppBuilder', 'Debug', '$(_NetCoreAppToolCurrent)', 'publish'))</WasmAppBuilderDir>
+    <WasmBuildTasksDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'WasmBuildTasks', 'Debug', '$(_NetCoreAppToolCurrent)', 'publish'))</WasmBuildTasksDir>
+    <MonoAOTCompilerDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'MonoAOTCompiler', 'Debug', '$(_NetCoreAppToolCurrent)'))</MonoAOTCompilerDir>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(RuntimeSrcDir)' == '' and '$(WasmBuildSupportDir)' != ''">
```

Comment:
Is it not possible to have both `RuntimeSrcDir` and `WasmBuildSupportDir` to be set? Should we relax this condition to just `'$(WasmBuildSupportDir)' != ''` if having `WasmBuildSupportDir` is intended to trump `RuntimeSrcDir`?

Reply:
Otherwise, having both be set would skip both of this and the prior `PropertyGroup`s

Reply:
With the new commit, we'll check, and complain if both are set. I want to have that restriction just to reduce pain for devs unsure about what precedence etc. This isn't really meant for external consumption yet.